assets/js/modules/analytics/datastore/properties.js

```diff
@@ -476,6 +476,10 @@ const baseSelectors = {
 	getPropertiesIncludingGA4: createRegistrySelector( ( select ) => ( state, accountID ) => {
 		let properties = select( STORE_NAME ).getProperties( accountID );
 
+		if ( ! properties ) {
+			return undefined;
+		}
```

Comment:
We also need to make sure that we return `undefined` when ga4 properties are being loaded too (when `ga4setup` feature is enabled).

Reply:
Done and added tests for this